File: 4-conditional-statements/cs.ex
defmodule Solution do
  def ifel(n) do
    cond do
      rem(n, 2) != 0 ->
        IO.puts("Weird")
      true ->
        cond do
          n<=5 ->
            IO.puts("Not Weird")
          n<=20 ->
            IO.puts("Weird")
          true ->
            IO.puts("Not Weird")
        end
    end
  end
end

{n,_} = IO.gets("")
    |> Integer.parse
Solution.ifel n

File: 4-conditional-statements/cs.erl
-module(solution).
-export([main/0]).



main() ->
    {N, _} = string:to_integer(string:chomp(io:get_line(""))),
        if
            (N rem 2) /= 0 ->
                io:fwrite("Weird");
            true -> 
                if
                    N=<5 ->
                        io:fwrite("Not Weird");
                    N=<20 -> 
                        io:fwrite("Weird");
                    true -> 
                        io:fwrite("Not Weird")
                end
        end.